File: 数学/chapter1/1.8/No1447/main.py
from typing import List
import math


class Solution:
    def simplifiedFractions(self, n: int) -> List[str]:
        ans = []
        for i in range(2, n + 1):
            for j in range(1, i):
                if math.gcd(i, j) == 1:
                    ans.append(str(j) + "/" + str(i))
        return ans


def main():
    sol = Solution()
    print(sol.simplifiedFractions(4))


main()

File: 数学/chapter1/1.8/No1447/main.cpp
#include <iostream>
#include <vector>
using namespace std;

class Solution {
public:
  vector<string> simplifiedFractions(int n) {}
};

signed main() {
  Solution sol;
  auto ans = sol.simplifiedFractions(4);
  for (auto &x : ans) {
    cout << x << ' ';
  }
  cout << endl;

  return 0;
}